app/test/ui/widgets/atr_preview_test.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:app/ui/widgets/atr_preview.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:intl/intl.dart';
+
+import '../../test_animal_transport_record.dart';
+
+void main() {
+  group('Animal Transport Record Preview', () {
+    testWidgets('Previews right information', (WidgetTester tester) async {
+      final testDate = DateTime.parse("2021-01-01 01:01");
+      final testCompany = "Test Company";
+      final testSpecies = "Cows";
+      final testRecWithDelvInfo = testAnimalTransportRecord().withDeliveryInfo(
```

Comment:
Note: We unfortunately cannot use Dart's fantastic cascade operator with the `with()` functions because they return an object.

Reply:
Also refactored test for easier reading (this is why I made the `with()` functions, better use them >:( )